shared/styles/tailwind.config.js

```diff
@@ -88,16 +90,15 @@ module.exports = {
         "200": "#F7F7F7",
         "100": "#F9F9F9"
       }
-     },
-     letterSpacing: {
-      tightest: '-.075em',
-      tighter: '-.05em',
-      tight: '-.025em',
-      normal: '0',
-      wide: '.025em',
-      wider: '.05em',
-      widest: '.1em',
```

Comment:
@slowbot Note widest was defined twice here, so I just guessed which one was correct so it would pass linting. Feel free to fix it as you see fit.